public/stylesheets/admin.css

```diff
@@ -106,3 +106,49 @@ li.active a, li.active a:hover {
 .tab.active {
   display:block;
 }
+
+.create-inventory {
+  display: flex;
+  justify-content: space-between;
+}
+
+form {
+  max-width: 100%;
+}
+
+.inventory-input {
+  font-family: 'Droid Sans', helvetica , sans-serif;
+  border: none;
+  outline: none;
+  font-size: 1rem;
+  text-indent: 5px;
+  width: 70%;
+  height: 2rem;
+  background: #ecf0f1;
+  -webkit-appearance: none;
```

Comment:
Consider using a [CSS post-processor](https://github.com/postcss/autoprefixer) to add browser specific prefixes, instead of manually coding them - this introduces the need for some additional tooling (like webpack), but really helps as your CSS codebase grows to focus on the display, instead of the tedium of cross browser CSS rules.